packages/components/src/index.js

```diff
@@ -12,6 +12,7 @@ export { default as ColorPalette } from './color-palette';
 export { default as ColorPicker } from './color-picker';
 export { default as Dashicon } from './dashicon';
 export { DateTimePicker, DatePicker, TimePicker } from './date-time';
+export { default as DimensionControl } from './dimension-control';
```

Comment:
I guess it would be good to export this component as __experimentalDimensionControl, until we test it in practice. It makes it easier to do the required API changes in the future.

Reply:
Good plan. Done.